src/lib/updates.js

```diff
@@ -17,27 +17,20 @@ export const createUpdate = async (files = [], channel, ts, user, text) => {
     ...files.map(async (file) => {
       const publicUrl = await getPublicFileUrl(file.url_private, channel, user);
       if (!publicUrl) {
-        await Promise.all([
-          react("remove", channel, ts, "beachball"),
-          react("add", channel, ts, "x"),
-          reply(channel, ts, t("messages.errors.filetype")),
-        ]);
-        return "error";
+        await reply(channel, ts, t("messages.errors.filetype"))
+        return;
       } else if (publicUrl.url.toLowerCase().endsWith("heic")) {
         await Promise.all([
-          react("remove", channel, ts, "beachball"),
-          react("add", channel, ts, "x"),
           postEphemeral(channel, t("messages.errors.heic"), user),
           app.client.chat.delete({
             token: process.env.SLACK_USER_TOKEN,
             channel,
             ts,
           }),
```

Comment:
I assume this should be moved, at the moment this is deleting the message from the channel if it has one `.heic` image

Reply:
Yeah I have no clue why that's there